src/Template/Element/index/table_actions.ctp

```diff
@@ -7,6 +7,15 @@ foreach ($actions['entity'] as $action) {
         continue;
     }
 
+    if ($action === 'delete') {
```

Comment:
Hardcoding this seems like a bad idea. Maybe we should add something to the action class that signifies that requests need to happen with `postLink`?